test/v2/core/sgETH.spec.js

```diff
@@ -53,4 +53,44 @@ describe("SgETH.sol", () => {
       .and.to.be.emit(sgEth, "RoleRevoked")
       .withArgs(ethers.constants.HashZero, deployer.address, deployer.address);
   });
+
+  it("e2e test", async () => {
+    // deploy sgeth
+    const WSGETH = await ethers.getContractFactory("WSGETH");
+    const wsgETH = await WSGETH.deploy(sgEth.address, 24 * 60 * 60);
+    await wsgETH.deployed();
+
+    const feeCalc = ethers.constants.AddressZero; // not sure if it is right
```

Comment:
iirc, theres 3 missing contracts here from the default deploy pipeline linked below: 

https://github.com/chimera-defi/SharedDeposit/blob/main/deploy/v2/lib/onchain_actions.js#L102

1. daoFeeSplitter -> OpenZeppellin vanilla paymentsSplitter , pulled in here https://github.com/chimera-defi/SharedDeposit/blob/ae40f847fcfa808186b680b240637ef16c29cd34/deploy/v2/lib/opts.js#L57
source: 
OZ-3.3. - https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v3.3/contracts/payment/PaymentSplitter.sol
Note:  - this is removed from OZ master, should check to make sure its still safe. Also need to rethink this build approach since it wasnt obvious to you what this contract should be?     

2. Withdrawals -> https://github.com/chimera-defi/SharedDeposit/blob/main/contracts/v2/core/Withdrawals.sol
3. RewardsReceiver -> https://github.com/chimera-defi/SharedDeposit/blob/main/contracts/v2/core/RewardsReceiver.sol

Reply:
Could you let me know parameters of each contract constructor?

Reply:
you can see the constructor args here
https://github.com/chimera-defi/SharedDeposit/blob/main/deploy/v2/lib/onchain_actions.js#L102
they are defined here https://github.com/chimera-defi/SharedDeposit/blob/main/deploy/v2/lib/opts.js#L45